src/components/ha-header-bar.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  customElement,
+  LitElement,
+  html,
+  unsafeCSS,
+  css,
+  property,
+} from "lit-element";
+// @ts-ignore
+import topAppBarStyles from "@material/top-app-bar/dist/mdc.top-app-bar.min.css";
+
+@customElement("ha-header-bar")
+export class HaHeaderBar extends LitElement {
+  @property({ type: Boolean }) centerTitle = false;
+
+  protected render() {
+    let title = html`<span class="mdc-top-app-bar__title">
+      <slot name="title"></slot>
+    </span>`;
+    if (this.centerTitle) {
+      title = html`<section
+        class="mdc-top-app-bar__section mdc-top-app-bar__section--align-center"
+      >
+        ${title}
+      </section>`;
+    }
+
+    return html`<header class="mdc-top-app-bar">
+      <div class="mdc-top-app-bar__row">
+        <section
+          class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start"
+          id="navigation"
+        >
+          <slot name="navigationIcon"></slot>
+          ${!this.centerTitle ? title : ""}
+        </section>
+        ${this.centerTitle ? title : ""}
+        <section
+          class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end"
+          id="actions"
+          role="toolbar"
+        >
+          <slot name="actionItems"></slot>
+        </section>
+      </div>
+    </header>`;
+  }
+
+  static get styles() {
+    return [
+      unsafeCSS(topAppBarStyles),
+      css`
+        .mdc-top-app-bar {
+          position: static;
```

Comment:
Are we sure we want to put this in the element itself and not have the parent decide on this?

Reply:
This could be used outside of dialogs, no?

Reply:
If you would want to use on the top scope you should use `<mwc-top-app-bar>`